spec/forms/providers/link_application/make_link_form_spec.rb

```diff
@@ -65,6 +70,16 @@
         expect(linked_application.confirm_link).to be false
       end
     end
+
+    context "when the answer is changed from family link to legal link" do
+      let(:linked_application) { build(:linked_application, associated_application_id: legal_aid_application.id, link_type_code: "FC_LEAD") }
+      let(:link_type_code) { "LEGAL" }
+
+      it "resets the linked_application model" do
```

Comment:
Is this test checking that changing the link type to family changes copy_case and copy_case_id to nil if they are already set? Would these not already be  set to nil before the test is run in any case? (i.e. would we not need something like `let(:legal_aid_application) { create(:legal_aid_application, copy_case: true, copy_case_id:  id_of_another_legal_aid_applicaton) }` to test the values are actually reset?

Reply:
I've rewritten this test which should be much better now